distributed/utils_test.py

```diff
@@ -1007,132 +1007,139 @@ def test_func(*outer_args, **kwargs):
             with clean(timeout=active_rpc_timeout, **clean_kwargs) as loop:
 
                 async def coro():
-                    with dask.config.set(config):
-                        s = False
-                        for _ in range(60):
-                            try:
-                                s, ws = await start_cluster(
-                                    nthreads,
-                                    scheduler,
-                                    loop,
+                    with tempfile.TemporaryDirectory() as tmpdir:
+                        config2 = merge({"temporary-directory": tmpdir}, config)
```

Comment:
nit: I'd prefer a more meaningful name like `merged_config`, but it really doesn't matter given the current scope of usage.